src/share/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>

```

Comment:
I wonder if we really need this file, or if you should just move these 2 links to the top-level `index.html`?

Reply:
Please dump this file.